data/languages/german.txt

```diff
@@ -682,19 +682,17 @@ Your skin
 no limit
 == Keine Begrenzung
 
-##### needs translation #####
-
 Borderless window
-== 
+== Randloses Fenster
 
 Handle rendering async from updates
-== 
+== Zeichne das Bild asynchron zu den Updates.
 
 Threaded rendering
-== 
+== Paralleles zeichnen des Bildes
```

Comment:
"Zeichnen" müsste groß geschrieben werden: ist in diesem Fall eine Substantivierung, weil wegen dem 'Parallel**es**' (= **das** parallele *Z*eichnen)